project/utility/testDoc.py

```diff
@@ -0,0 +1,70 @@
+import _jpype
```

Comment:
development stub?

Reply:
Used on line 54.   (I can fix it once JPackage isinstance logic goes in)